"Add post open-access-peer-review" (#32)
ashwinvis/ashwinvis.github.io: content/2020-05-23-open-access-peer-review.rst

Say yes to open-access + peer-review
####################################

:author: Ashwin Vishnu Mohanan
:date: 2020-05-23T20:59:28.194153
:slug: open-access-peer-review
:status: published
:summary: The only model for peer-review which makes sense: the "No free view? No review!" signature campaign and why the existing publishing system needs an overhaul
:category: Blog
:tags: research, open-access, academia

.. :cover: https://cdn.pixabay.com/photo/2017/03/03/13/56/key-2114046_960_720.jpg

There are fundamentally several problems with the current publishing model of
scientific articles:

- Authors toil and work polishing nearly every word of the article
- Reviewers volunteer out of goodwill or to keep in touch with research
- Editors coordinate the whole effort
- Libraries or businesses pay to get access to and distribute the end result

And no-one above in this process gets paid. High charges do not make sense in

> the electronic age, where fewer article get actually printed into hard copy.

Author: ```suggestion
the electronic age, when fewer articles get actually printed into hard copy.
```

Here is a documentary describing why this system is ridiculous at best and
predatory at worst.

.. raw:: html

   <iframe width="560" height="315" sandbox="allow-same-origin allow-scripts allow-popups" src="https://peertube.social/videos/embed/75ba36d5-1e81-4db4-95b0-8dd4790eb282" frameborder="0" allowfullscreen></iframe>

A paradigm shift
-----------------

Plan-S_ and Bibsam consortium made a huge dent in this process. This movement gave
libraries and universities the ability and courage to collectively bargain and
walk out of the current subscription model with exorbitant prices. This was a
top-down change and resulted in new deals being made to:

- benefit authors to make new articles open-access without the authors
  paying a hefty Application Processing Charge (APC), but rather the university
  as a whole
- make certain journals open-access or hybrid

What we need now is a grassroots movement, a bottoms-up approach. As authors, we
should be willing to share our research at different stages to preprint servers
(for example, arXiv) and similar outlets (Zenodo, Figshare etc.). The
Sherpa-Romeo_ portal is my go-to tool in finding out what kind of article can

> be archived for public-consumption. Also check with your library to find out

Author: ```suggestion
be archived for public-consumption. Also, check with your library to find out
```

if they could cover the APC charges.

And as potential reviewers, this campaign_ is relevant:

.. image:: /static/nfvnr-button.svg
   :target: campaign_
   :alt: No free view? No review!
   :width: 50%

> I think it is our turn as researchers and academics to say no to articles
> which would end up behind a paywall. It is just not ethical in today's world.

Author: ```suggestion
I think it is our turn as researchers and academics to say no to review articles
which would end up behind a paywall. It is simply not ethical in today's world.
```
